plugins/destination/s3/client/write.go

```diff
@@ -24,13 +24,26 @@ func (c *Client) WriteTable(ctx context.Context, msgs <-chan *message.WriteInser
 	var s *filetypes.Stream
 
 	for msg := range msgs {
-		if s == nil {
-			table := msg.GetTable()
+		fileClient := c.Client
+		table := msg.GetTable()
 
+		if s == nil {
+			if _, ok := c.objectKeys[table.Name]; !ok {
+				c.objectKeys[table.Name] = []string{}
+			}
 			objKey := c.spec.ReplacePathVariables(table.Name, uuid.NewString(), time.Now().UTC())
+			c.objectKeys[table.Name] = append(c.objectKeys[table.Name], objKey)
+
+			if table.Name == "cloudquery_sync_summary" {
```

Comment:
This seems too tightly coupled with the CLI that uses the same name for the table

Reply:
It is designed to be an extension of the CLI summary table, all this functionality is doing is adding a column to that table.